docs/installation/autoware/docker-installation.md

```diff
@@ -1,106 +1,161 @@
-# Docker installation
+# Open AD Kit: Containerized Workloads for Autoware
+
+Open AD Kit offers two types of Docker image to let you get started with Autoware quickly: `devel` and `runtime`.
+
+1. The `devel` image enables you to develop Autoware without setting up the local development environment.
+2. The `runtime` image contains only runtime executables and enables you to try out Autoware quickly.
 
 !!! info
 
-    Since this page explains Docker-specific information, it is recommended to see [Source installation](./source-installation.md) as well if you need detailed information.
+    Before proceeding, confirm and agree with the [NVIDIA Deep Learning Container license](https://developer.nvidia.com/ngc/nvidia-deep-learning-container-license). By pulling and using the Autoware Open AD Kit images, you accept the terms and conditions of the license.
 
-Here are two ways to install Autoware by docker:
+## Prerequisites
 
-- The first way is to start Autoware with `prebuilt image`, this is a quick start, this way you can only run Autoware simulator and not develop Autoware, it is only suitable for beginners
+- Docker
+- NVIDIA Container Toolkit (preferred)
+- NVIDIA CUDA 12 compatible GPU Driver (preferred)
 
-- The second way is to start Autoware with `devel image`, which supports developing and running Autoware using docker
+The [setup script](../setup-dev-env.sh) will install all required dependencies with the setup script:
 
-## Docker installation for quick start
+```bash
+./setup-dev-env.sh -y docker
+```
 
-[docker installation for quick start](./docker-installation-prebuilt.md)
+To install without **NVIDIA GPU** support:
 
-![type:video](https://youtube.com/embed/3KUhEFkEbI8)
+```bash
+./setup-dev-env.sh -y --no-nvidia docker
+```
 
-## Docker installation for development
+!!! info
 
-[docker installation for development](./docker-installation-devel.md)
+    GPU acceleration is required for some features such as object detection and traffic light detection/classification. For details of how to enable these features without a GPU, refer to the [Running Autoware without CUDA](../how-to-guides/others/running-autoware-without-cuda.md).
```

Comment:
Dead link here (I had raised that as a general comment instead of a review https://github.com/autowarefoundation/autoware-documentation/pull/516#pullrequestreview-1914254273)

Reply:
Fixed with https://github.com/autowarefoundation/autoware-documentation/pull/516/commits/6aa1a9a2ce6030a087e92b7a8941eca81891c5de

And another one in https://github.com/autowarefoundation/autoware-documentation/pull/516/commits/7b45d8b248a66df52c47c517b526f1c2d130e488

The rest of dead links are from other places:
- https://github.com/autowarefoundation/autoware-documentation/actions/runs/8175681821/job/22353360585?pr=516#step:3:514

Thanks for noticing.